### ISSUE TITLE
Website: web demo does not show certain diagnostics

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#1033]
e.g.
```javascript
var i a test = "test"; // should error, but web demo doesn't
console.log(i);
```
https://quick-lint-js.com/demo/

**Comment by strager** [comment]
Fixed in commit 1190a5b40f4a73ae4b81f843aed353442e1eaa80.
